tmt-app/src/Components/CareerPage.jsx

```diff
@@ -24,6 +55,54 @@ function CareerPage() {
       >
         {data.name} (Career)
       </Typography>
+      <Grid
+        container
+        spacing={4}
+        justifyContent="center"
+        sx={{ paddingX: "1rem" }}
+      >
+        {sectionInfo.map((item, index) => {
+          return (
+            <Grid item key={index} xs={12} md={6}>
+              <Card
+                sx={{
+                  boxShadow: "none",
+                  backgroundColor: `${item.color}`,
+                  height: "100%",
+                  boxSizing: "border-box",
+                }}
+              >
+                <Typography variant="h3" sx={{ paddingBottom: "1.5rem" }}>
+                  {item.title}
+                </Typography>
+                <Typography variant="body">
+                  {Array.isArray(item.info) ? (
+                    <List
+                      sx={{
+                        listStyleType: "disc",
+                        paddingLeft: "1.75rem",
+                      }}
+                    >
+                      {item.info.map((item, index) => {
+                        return (
+                          <ListItemText
+                            key={`${item.name}-${index}`}
+                            sx={{ display: "list-item" }}
+                          >
+                            <Typography variant="body">{item}</Typography>
+                          </ListItemText>
```

Comment:
This component has some props we might be able to leverage as well, is there any difference/benefit to the props they offer as an alternative to using the children slot?

Reply:
I will take another look at the props for this component and make the change